rust/libsettings-sys/libsettings_wrapper.h

```diff
@@ -0,0 +1,52 @@
+#ifndef LIBSETTINGS_WRAPPER_H
+#define LIBSETTINGS_WRAPPER_H
+
+typedef unsigned char uint8_t;
```

Comment:
Since this is working I don't want to throw a wrench into things, but we might be able to dump this file later and follow the pattern established here: https://github.com/swift-nav/swiftnav-rs/blob/master/swiftnav-sys/build.rs#L62